site/gatsby-site/src/components/submissions/schemas.js

```diff
@@ -98,8 +114,16 @@ export const schema = yup.object().shape({
     .optional()
     .nullable(),
   incident_ids,
-  incident_date,
-  incident_title: yup.string().nullable(),
+  incident_date: yup.string().when('incident_ids', {
+    is: (incident_ids) => !incident_ids || incident_ids.length == 0,
+    then: yup.string().required('*Incident Date is required'),
+    otherwise: yup.string().nullable(),
+  }),
+  incident_title: yup.string().when('incident_ids', {
+    is: (incident_ids) => !incident_ids || incident_ids.length == 0,
+    then: yup.string().required('Incident title is required'),
```

Comment:
I'm not familiar with yup. Does this string also need a "*" asterisk in the beginning, like the others?